### PR TITLE
show a message for empty feeds

### PR DESCRIPTION
**Pull request by p0o**
This PR should show a message for empty feeds

### CODE REVIEW

**Comment by adcpm** [comment]
Actually the "empty feed message" will appear on Bookmark and user profile too. We need a way to handle custom message for these pages.

**Comment by p0o** [comment]
@adcpm shouldn't be a problem. What kind of messages we need for these pages?

**Comment by adcpm** [comment]
There is already an empty message on bookmark page, you can keep same text. About user profile i would see a different message if you its your profile you are visiting or if its another user profile. My profile the message could be : "You didn't publish any story yet. Start now" and another user "This user don't have any story published yet."

**Comment by p0o** [comment]
@adcpm should work as expected

File: src/user/UserFeed.js [hunk 1]
```diff
@@ -30,14 +31,20 @@ export default class UserProfileFeed extends Component {
...
     return (
-      <Feed
-        content={content}
-        isFetching={isFetching}
-        hasMore={hasMore}
-        loadContent={loadContentAction}
-        loadMoreContent={loadMoreContentAction}
-        route={this.props.route}
-      />
+      <di>
```
Comment (nil1511) [inline src/user/UserFeed.js]: I think It should be div. 😄

File: src/feed/Feed.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
```
Comment (nil1511) [inline src/feed/Feed.js]: This file changes can be cleared.

File: src/user/UserFeed.js (hunk 1 above)
Comment (p0o) [inline src/user/UserFeed.js]: @nil1511 Hahaha, right 😄

**Comment by p0o** [comment]
@nil1511 thank you, should be okay now.

**Comment by adcpm** [comment]
@p0o Empty message is visible on user profile while posts are fetching.

**Comment by p0o** [comment]
@adcpm thank you. It's fixed now.

**Pull request merged**